Source/Scene/Globe.js

```diff
@@ -650,7 +645,14 @@ define([
             return undefined;
         }
 
-        return ellipsoid.cartesianToCartographic(intersection, scratchGetHeightCartographic).height;
+        var height = ellipsoid.cartesianToCartographic(intersection, scratchGetHeightCartographic).height;
+
+        // For low-detail tiles, large triangles often cut the the globe and appear to be at a much
+        // lower height than actually makes any sense. So clamp the height to the actual height range
+        // of the tile.
```

Comment:
Is this the issue where you can see stark elevation differences between neighbouring tiles while the terrain is loading in? Is there an easy way to recreate this or see this clamping height in action?

Reply:
Originally, terrain clamping didn't work at all in this branch. I added this thinking it would help, but it didn't, and I had to add the `_neededPositions` system instead. So we could probably remove it. 

It's a pretty reasonable change, though. If a terrain tile is known to contain heights beween -100 and 8700 meters, and you ask for a height at a point within that tile, it seems reasonable that it wouldn't return something less than -100. Without this check, level 0 tiles will commonly return some very negative numbers.

On the other hand, _without_ this change, the returned height will actually be on the tile's triangles. With this change, it will sometimes be floating above them (even if imperceptibly because of the SSE-driven selection).

So ¯\_(ツ)_/¯ . Anyone have a strong opinion?